README.md

```diff
@@ -102,13 +102,16 @@ assert(tuple5 === #[2, 2, 3, 4]);
 #### Computed access
 
 ```js
+const k = "b";
 const record = #{ a: 1, b: 2, c: 3 };
-const tuple = #[1, 2, 3, 4];
+assert(#{ ...record, [k]: 5 } === #{ a: 1, c: 3, b: 5 });
```

Comment:
This is a bit misleading, since record keys are sorted. I'd write this as `#{ a: 1, b: 5, c: 3 }`